INFO - 27/11/2024 00:04:28 : Index leak_model already exists, skipping creation. - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:175)
INFO - 27/11/2024 00:04:28 : Index generic_model already exists, skipping creation. - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:184)
ERROR - 27/11/2024 00:10:29 : ELASTIC 1 : Initialization failed: Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7f29973cb950>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:187)
ERROR - 27/11/2024 00:10:31 : ELASTIC 1 : Initialization failed: Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7f88d20a2810>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:187)
ERROR - 27/11/2024 00:10:31 : ELASTIC 3 : [5] Something unexpected happened while reading : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7f88d20b7190>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:204)
ERROR - 27/11/2024 00:13:49 : ELASTIC 1 : Initialization failed: Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7f79314afcd0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:187)
INFO - 27/11/2024 00:14:16 : Index leak_model already exists, skipping creation. - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:175)
INFO - 27/11/2024 00:14:16 : Index generic_model already exists, skipping creation. - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:184)
ERROR - 27/11/2024 00:16:07 : ELASTIC 1 : Initialization failed: Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55dc9d590>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:187)
ERROR - 27/11/2024 00:16:07 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55dc7de10>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:07 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55dc9c890>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:07 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55dc61110>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:07 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55de1e490>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:07 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55dc61290>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:16 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55ddd4750>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:16 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55ddc9b50>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:16 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa5598b0190>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:16 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a86e210>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:16 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55dc5bad0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:16 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55ddc6ed0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:17 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55dc609d0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:18 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a86df90>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:18 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a86dad0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:19 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55ddc7350>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:20 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55ddf8c90>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:20 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a867d50>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:21 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a86c6d0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:21 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55ddc9950>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:22 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a843750>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:23 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a850f10>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:23 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a864390>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:24 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55ddc79d0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:24 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a864350>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:24 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a853910>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:25 : ELASTIC 1 : Initialization failed: Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7f69fdca7b90>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:187)
ERROR - 27/11/2024 00:16:25 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a84ab90>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:27 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a8530d0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:27 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a840150>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:27 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a867750>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:28 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a853cd0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:29 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a84bbd0>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:29 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a864e10>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:30 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a849810>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
ERROR - 27/11/2024 00:16:31 : [1] Something unexpected happened while inserting : Connection error caused by: ConnectionError(Connection error caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x7fa55a850a10>: Failed to establish a new connection: [Errno 111] Connection refused)) - elastic_controller (/app/app_manager/elastic_manager/elastic_controller.py:246)
